_taxonomy.rst

```diff
@@ -0,0 +1,69 @@
+.. _taxonomy:
+
+Documentation Taxonomy
+======================
+
+Data Management uses and produces multiple classes of documentation.
+Each class fulfills a specific role, with a consistent management and technical process.
+
+.. _taxonomy-outline:
+
+Document classes
+----------------
+
+The classes of DM documentation are:
+
+Requirements and interface control documentation (LPM, LSE)
+   These documents specify functionality that the Data Management System must deliver to the Project.
+   Similarly, **interface control documents** are agreements between subsystems on functionality that cross subsystem boundaries.
+   The production of this documentation is described in `LPM-19: Change Control Process`_ and its management is described in `LPM-51: Document Management Plan`_.
+
+Change-controlled DM design documentation (LDM)
+   These documents define the **scope** and **budget** of Data Management work that meet the **requirements** and **interface control documentation**.
+   DM work is expected to be consistent with these **design documents**.
+   LDM document authors are responsible for updating design documentation whenever necessitated by revised project requirements, or by evolving implementation choices.
+   Design documents must be approved by the Change Control Board (`LPM-19: Change Control Process`_) before becoming the new baseline.
+   These documents are also governed by `LPM-51: Document Management Plan`_.
+   See :ref:`ldm` for how change-controlled documents are handled by the Data Management Documentation Architecture.
+
+Technical notes (DMTN, SQR)
+   Technical notes are a standardized, durable, document series allowing useful information to be published on demand by any Data Management team member wishing to capture and share knowledge.
```

Comment:
We may need another group for things like test plans and verification documentation.  I'm not sure that DMTNs would be adequate for this.

It's also not clear where lower-level, non-change-controlled design documentation fits.  Some of this should be with the code, but not necessarily in user guides (although maybe this is really what "developer guides" means).